umap/static/umap/js/modules/importers/cadastrefr.js

```diff
@@ -0,0 +1,62 @@
+import { DomUtil } from '../../../vendors/leaflet/leaflet-src.esm.js'
+import { BaseAjax, SingleMixin } from '../autocomplete.js'
+import * as Util from '../utils.js'
+import { AutocompleteCommunes } from './communesfr.js'
+
+const TEMPLATE = `
+  <h3>Cadastre</h3>
+  <p>Importer le cadastre d'une commune française..</p>
+  <select name="theme">
+    <option value="batiments">Bâtiments</option>
+    <option value="communes">Communes</option>
+    <option value="feuilles">Feuilles</option>
+    <option value="lieux_dits">Lieux dits</option>
+    <option value="parcelles" selected>Parcelles</option>
```

Comment:
I'm tempted to put the selected choice at the top of the list, it is a more classic pattern (and I get that it's sorted alphabetically for now but still).

Reply:
I'm tempted to think that given the list is small, the alphabetical order is more natural and convenient once open. Let's try that way and see ?